ocaml/testsuite/tests/typing-layouts-err-msg/value.ml

```diff
@@ -268,7 +268,8 @@ Error: This expression has type ('a : value)
        The layout of 'b is void
          because of the annotation on the type variable 'b.
        But the layout of 'b must overlap with value
-         because it is or unifies with an unannotated universal variable.
+         because it's a fresh unification variable,
+         defaulted to layout value.
```

Comment:
I think this error message should probably be improved. The reason of this history should mention that it comes from an unannotated universal variable. I don't know if reasons carry locations, but ideally it would point at the relevant location too.